README.md

```diff
@@ -161,6 +161,7 @@ logging config format for content details.
 
 -  Volume metadata file got deleted while removing volume from VM(placed on Esx2) which is in use by another VM(placed on Esx1) [#1191](https://github.com/vmware/docker-volume-vsphere/issues/1191). It's an ESX issue and will be available in the next vSphere release.
 -  Full volume name with format like "volume@datastore" cannot be specified in the compose file for stack deployment. [#1315](https://github.com/vmware/docker-volume-vsphere/issues/1315). It is a docker compose issue and a workaround has been provided in the issue.
+-  Volume creation through vDVS by specifying datastore cluster name rather than a datastore name is not supported. Datastore clusters (as a part of Storage DRS) are a VC feature and are not available on individual ESX. [#556](https://github.com/vmware/docker-volume-vsphere/issues/556)
```

Comment:
Specifying "Datastore Cluster" name during Volume creation is not supported. Datastore clusters (as a part of Storage DRS) is a VC feature and not available on individual ESX. [#556](https://github.com/vmware/docker-volume-vsphere/issues/556)